terraform/environments/analytical-platform-compute/helm-charts-actions-runners.tf

```diff
@@ -69,6 +69,32 @@ resource "helm_release" "actions_runner_mojas_create_a_derived_table" {
   ]
 }
 
+resource "helm_release" "actions_runner_mojas_create_a_derived_table_non_spot" {
+  count = terraform.workspace == "analytical-platform-compute-production" ? 1 : 0
+
+  /* https://github.com/ministryofjustice/analytical-platform-actions-runner */
+  name       = "actions-runner-mojas-create-a-derived-table-non-spot"
+  repository = "oci://ghcr.io/ministryofjustice/analytical-platform-charts"
+  version    = "2.320.0-2"
+  chart      = "actions-runner"
+  namespace  = kubernetes_namespace.actions_runners[0].metadata[0].name
+  values = [
+    templatefile(
+      "${path.module}/src/helm/values/actions-runners/create-a-derived-table/values.yml.tftpl",
+      {
+        github_organisation  = "moj-analytical-services"
+        github_repository    = "create-a-derived-table"
+        github_runner_labels = "analytical-platform-non-spot"
+        eks_role_arn         = "arn:aws:iam::593291632749:role/create-a-derived-table"
```

Comment:
Please, use the local for the number, non-member accounts are in that list too. :D